spec/_plugins/image_helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'jekyll'
+require 'jekyll/tagging'
+require_relative '../../src/_plugins/image_helper.rb'
+
+describe 'ImageTag' do
+
+    it 'Can render a simple image with alt text' do
+      token = double
+      expect(token).to receive :line_number
+
+      tag = Jekyll::ImageTag.send :new, 'img', '/assets/custom/img/blog/ncraft-2017/event-storming-with-brandolini.jpg "ALT"', token
+
+      expect(tag.render(nil)).to eq('<p></p><img src="/assets/custom/img/blog/ncraft-2017/event-storming-with-brandolini.jpg"  alt="ALT" title="ALT" class="img img-center img-fluid style-screengrab">')
+    end
+    
+    it 'will throw an exception when an img does not have a non-blank alt tag' do
```

Comment:
double negation here "does not have a non-blank alt"
What do you think about this? 
"will throw an exception when an img has a blank alt tag"

Reply:
Renamed.